docs/cep/tutorials/using_rest_api.md

```diff
@@ -178,234 +178,232 @@ print("\nStream App Deleted: ", result)
-        const deletion = await connection.req(
-          `/_fabric/_system/_api/streamapps/${STREAM_APP_NAME}`,
-          {
-            method: "DELETE",
-          }
-        );
-
-        console.log("STREAM APP DELETED SUCCESSFULLY", deletion);
-      } catch (e) {
-        console.error(e);
+    await connection.req(
+      `/_fabric/_system/_api/streamapps/${streamAppName}/active?active=true`,
+      {
+        method: "PATCH"
       }
+    );
+
+    console.log("Activating stream app...", streamAppName);
+
+    await new Promise((resolve) => setTimeout(resolve, 10000));
+
+    console.log("Stream app activated successfully");
+
+    /* ------------------ Publish messages to Sample StreamApp ------------------ */
+    const region = isGlobal ? "c8global" : "c8local";
+    const streamName = `${region}s.${stream}`;
+
+    // Fetching local URL in case the stream is local (which is defined in the streamapp)
+    const localDcDetails = await connection.req(`/datacenter/local`, {
+      method: "GET"
+    });
+
+    const dcUrl = localDcDetails.tags.url;
+
+    const url = isGlobal
+      ? federationName
+      : `api-${dcUrl}`;
+
+    const otpProducer = await connection.req(`/apid/otp`, {
+      method: "POST"
+    });
+
+    const producerUrl = `wss://${url}/_ws/ws/v2/producer/persistent/${tenant}/${region}._system/${streamName}?otp=${otpProducer.otp}`;
+
+    /* -------------------------- Initalizing Producer -------------------------- */
+
+    const producer = new WebSocket(producerUrl);
+
+    producer.onopen = function () {
+      console.log("WebSocket:Producer is open now for " + streamName);
+    };
+
+    producer.onerror = function () {
+      console.log(
+        "Failed to establish WebSocket:Producer connection for " + streamName
+      );
     };
 
-    run();
+    producer.onclose = function () {
+      console.log("Closed WebSocket:Producer connection for " + streamName);
+    };
+
+    producer.onmessage = function () {
+      console.log("WebSocket:Producer message sent successfully");
+    };
+
+    await new Promise((resolve) => setTimeout(resolve, 10000));
+
+    const inputData = [
+      {
+        deviceID: "AD11",
+        roomNo: 200,
+        temperature: 18
+      },
+      { deviceID: "AD11", roomNo: 201, temperature: 47 }
+    ];
+
+    producer.send(
+      JSON.stringify({
+        payload: Buffer.from((JSON.stringify(inputData[0]))).toString('base64')
+      })
+    );
+
+    await new Promise((resolve) => setTimeout(resolve, 10000));
+
+    producer.send(
+      JSON.stringify({
+        payload: Buffer.from((JSON.stringify(inputData[1]))).toString('base64')
+      })
+    );
+
+    await new Promise((resolve) => setTimeout(resolve, 10000));
+
+    producer.close();
+
+    /* ----------------------------- Verify results ----------------------------- */
+
+    const selectQuery = "FOR doc IN tutorialAppOutputTable return doc";
```

Comment:
Is this a C8QL? If so, shouldn't RETURN be all caps?

Reply:
c8ql supports caps and lowercases both actually

Reply:
anyways I will make it caps for consistency

Reply:
done